eng/pipelines/handlers.yml

```diff
@@ -49,6 +49,7 @@ schedules:
 
 variables:
   - template: /eng/pipelines/common/variables.yml
+  - template: templates/common/vs-release-vars.yml@sdk-insertions
```

Comment:
we need to start thinking if we need these repos closed, or if they can be open. This will allow us to move to other azdo instance for example like dnceng that have strick policies related with external private references.